pkg/steps/pod.go

```diff
@@ -48,8 +49,9 @@ func (s *podStep) Inputs(ctx context.Context, dry bool) (api.InputDefinition, er
 }
 
 func (s *podStep) Run(ctx context.Context, dry bool) error {
-	log.Printf("Executing %s %s", s.name, s.config.As)
-
+	if !s.config.SkipLogs {
```

Comment:
This smells bad, why isn't this going to make for inscrutable logs that admins hate?

Reply:
There are pod steps called within jobs (where you want artifacts).  The log output when you call these nested logs is more confusing - the step for "build release image" doesn't need to show `executing pod release-latest` because it already said what it is doing.  We still print pod logs when they fail which is 95% of all debugging.

Reply:
The import from initial needs to run two pods, but the pods should never fail and the output of the nested pod within the import step is very confusing when reading it.  Only the top most step matters in terms of you needing to understand what is happening.

Reply:
Added a long godoc on PodStepConfiguration to explain this, let me know if that helps clarify.